### ISSUE TITLE
Connection Error for Report Loader

### ISSUE THREAD

**Issue by HariWu1995** [phamdinhkhanh/vnquant#21]
Hi,

Can you help me to check API of VNDirect to crawl report? I cannot run any of FinanceLoader's functions, including:
- get_basic_index()
- get_finan_report()
- get_business_report()
- get_cashflow_report()

The common error is 
`requests.exceptions.ConnectionError: HTTPSConnectionPool(host='finfo-api.vndirect.com.vn', port=443)...(Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001B1F389F970>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))`

**Comment by HariWu1995** [comment]
I have tested DataLoader and ran into the same problem! 
Maybe VNDirect changed its APIs or something?

**Comment by HariWu1995** [comment]
I have just used Kaggle to run and it succeeded. Maybe my API is locked?

**Comment by Griffs2** [comment]
I meet this issue. Did anyone face the same problem?

SSLError: HTTPSConnectionPool(host='finfo-api.vndirect.com.vn', port=443): Max retries exceeded with url: /v4/ratios?q=code:MBB~itemCode:53030,52005,51050,53021,52001,52002,54018,712010,712020,712030,712040~reportDate:2022-12-31 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1129)')))

**Comment by HariWu1995** [comment]
> I meet this issue. Did anyone face the same problem?
> 
> SSLError: HTTPSConnectionPool(host='finfo-api.vndirect.com.vn', port=443): Max retries exceeded with url: /v4/ratios?q=code:MBB~itemCode:53030,52005,51050,53021,52001,52002,54018,712010,712020,712030,712040~reportDate:2022-12-31 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1129)')))

If you follow groups on FB for VN stock market, you'll know VNDirect crashed yesterday. Some raised issue of SSL Certification out of date.

**Comment by phamdinhkhanh** [comment]
Dear all, now this issue is fixed according to release version 0.1.0

```
import vnquant.data as dt
loader = dt.FinanceLoader('VND', '2019-06-02','2020-12-31', data_source='VND', minimal=True)
data_basic = loader.get_basic_index()
print(data_basic.head())
data_basic.head().to_html('data_basic.html')

```
